lib/transport_web/templates/page/user_organizations.html.eex

```diff
@@ -0,0 +1,29 @@
+<h1><%= gettext "first_step" %></h1>
+<h2><%= gettext "declare_organization" %></h1>
+<%= if @has_organizations do %>
+
+<div class="userorganizations">
+<%= for organization <- @organizations do %>
+    <div>
+        <div class="organizationimage">
+          <img src=<%= organization["logo"] %>>
+        </div>
+        <div class="organizationdetails">
+          <%= organization["name"] %>
+        </div>
+        <div>
+            <a href="mailto:contact@transport.data.gouv.fr" class="button" role="link">
```

Comment:
```elixir
assigns[:contact_email]
```

?

Reply:
not really useful, this will be erase by the next PR.